tests/unit_tests/docmaps/v2/codecs/docmaps_test.py

```diff
@@ -563,25 +577,25 @@ def test_should_populate_assertions_for_second_manuscript_published_step(self):
             'status': 'manuscript-published'
         }]
 
-    def test_should_populate_inputs_for_vor_published_step(self):
+    def test_should_populate_inputs_for_vor_published_step_for_opt_ins(self):
+        manuscript_versions = [
+            {**MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
```

Comment:
minor: might be easier to read if it was all on one line or curly brackets on separate lines?

i.e. one of the following

```python
manuscript_versions = [{
    **MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
    'vor_publication_date': VOR_PUBLICATION_DATE_1
}]
```

Or

```python
manuscript_versions = [
    {
        **MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
        'vor_publication_date': VOR_PUBLICATION_DATE_1
    }
]
```

Or

```python
manuscript_versions = [
    {**MANUSCRIPT_VERSION_WITH_EVALUATIONS_1, 'vor_publication_date': VOR_PUBLICATION_DATE_1}
]
```

What do you think?